### ISSUE TITLE
Go install fails

### ISSUE THREAD

**Issue by farzadmf** [sachaos/viddy#36]
Hi, I tried to install this using `go install github.com/sachaos/viddy@latest`, and it's failing with this error :slightly_frowning_face: 

```
go install: github.com/sachaos/viddy@latest (in github.com/sachaos/viddy@v0.3.3):
        The go.mod file for the module providing named packages contains one or
        more replace directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

**Comment by sachaos** [comment]
@farzadmf I've released 0.3.4 and fix the bug. Thank you for your bug report!

**Comment by farzadmf** [comment]
Yup, thank you @sachaos for the quick action, I saw your PR and tried with `go install ...@latest`, and it's working fine now

And ... thank you for the amazing tool, LOVE the idea and concept, and kudos to you for coming up with the idea. Keep up the good work
